Minor edits to Vite + React quickstart (#6926)
aws-amplify/docs: src/fragments/gen2/quickstart/build-a-backend.mdx

### Add data

> `create-amplify` provides the scaffolding of a `/amplify/data/resource.ts` file, which is ready to deploy.

Reviewer: as this fragment comes together it is a bit back and forth on the preceding slash when referencing files 
![image](https://github.com/aws-amplify/docs/assets/5033303/74dba362-be0c-4f77-b0cc-fce70b49ef25)
![image](https://github.com/aws-amplify/docs/assets/5033303/dee648f8-64f0-4271-a846-7f0712364b75)

I believe this was added to signify the project root, but this can be confusing to those newer to the space as this is typically aliased to `~` or `@` (depending on the framework) and these references are off the filesystem root. Can we remove the preceding slash from the file references?

<Accordion title="See the complete /amplify/data/resources.ts">

> If you open the `/amplify/data/resource.ts` file in your text editor, you will see some generated code already there.

Reviewer 2: A suggestion to make this a statement instead of starting with a question "If you..." and update the language to accurately describe what's in the file

```suggestion
Open the `/amplify/data/resource.ts` file in your text editor, and you will see a default data model generated for you.
```
